public/scripts/countryData.js

```diff
@@ -0,0 +1,90 @@
+(function(module) {
+  var origin = {};
+  origin.all = [];
+  origin.winner = '';
+  origin.diff = 1000000;
+  var guatemala = {};
+  var nicaragua = {};
+  var honduras = {};
+  guatemala.all = [];
+  nicaragua.all = [];
+  honduras.all = [];
+  guatemala.gs = [];
+  nicaragua.gs = [];
+  honduras.gs = [];
+  guatemala.diff = 0;
+  nicaragua.diff = 0;
+  honduras.diff = 0;
+
+  origin.getData = function() {
+
+    $.ajax(
```

Comment:
Reminder: `$.ajax()` is a low-level jQuery method that you actually don't need. You could have done this more simply by using `$.get()`, and then chaining your `.done()` and `.error()`.